### ISSUE TITLE
Revisit design language of default tools

### ISSUE THREAD

**Issue by gentlecolts** [Yellow-Dog-Man/Resonite-Issues#80]
### Describe the bug?

The current default tools do a lot of really great things, however I feel like there's a few issues with them that could use a second pass.  All of the tools have a distinct shape, however I question how well this reads from a distance.  Their shape's unique and readable, but their design is somewhat complex and "thin", which I feel reduces the ability to understand them from a distance and over-complicates their design.  I also think their use of color is quite lacking and contributes to the lack of "identity" that each tool seems to have.  The colors given to them in the inventory are an awesome tool for organization, but as it currently stands, most of the tools seem to read as "black and white metal-y doodads", with a large portion of their visual real-estate taken up by glass orb, shiny silver, etc, even if some have some distinct accents.

### To Reproduce

Put yourself in the position of someone who wants to draw fanart of the game, maybe make icons, quick doodles that need to make their intent clear with a small amount of information, or put yourself in the position of someone who's trying to tell a new user which item to spawn from their inventory.  We can't just say "oh spawn out the yellow one", we need to be more descriptive than that.

### Expected behavior

I think there's something good we have to work with already present, but I'm not quite experienced enough of a designer to make firm suggestions.  Some observations, however:

- the dev tool has a distinct "red" to its tip, perhaps we could use more red across the tool, make it "the red tool"
- the protoflux tool has a very nice purple as its overall look, i wouldn't change this, in fact more tools could be like this
- The white/gold look of the component clone is a good base, but again we can lean into that identity more

In fact, especially on that last note, I think a lot of why these tools that already DO have good colorschemes lose their identity is because a) they're too shiny, and distinct areas of color are lost in reflections and lighting effects, perhaps a more plastic-y look, or even a "rough metal" could be a better direction, and b) the areas of distinct color tend to be too thin, not occupying enough visual area

Bottom line here: We need shapes AND colors of each tool to be clear, readable, and unique, including factoring in how non-color maps factor into the models.

### Additional Context

Very much open to discussion here, I feel like this is not going to be a quickly solved issue

### Reporters

zangooseoo

**Comment by AegisTheWolf** [comment]
### To give context on the designs of the tools:

- The current tools have been designed to have unique silhouettes to not just rely on color (as color blindness is something to account for) 

- The basic concept of the tools focus on a "ball and socket" design for a multi-tool and to be used on its own, additionally these tools are based off of real tools when feasible or software based tools and iconography made physical/grounded in a sense.

- Originally all the tools started the same color but I added color to the primary color parts of each tool when possible.

- The tools also had to go with the consistent design style for the art direction we were going for as well with a sci-fi/fantasy blend to the tools.

This feedback definitely can be referenced for a future additional set of tools to be made alongside the original, this does not stop anyone else from making a set of tools to share with the community on their own.
